Packs/GitHub/Integrations/GitHub/GitHub.yml

```diff
@@ -2815,6 +2815,37 @@ script:
     - contextPath: GitHub.File.SHA
       description: The SHA of the file.
       type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: organization
+      description: The name of the organization.
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: team_slug
+      description: Team name.
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '30'
+      isArray: false
+      name: maximum_users
+      description: Miximum number of users to return.
+      required: false
+      secret: false
+    deprecated: false
+    description: List team members
+    execution: false
+    name: GitHub-list-team-members
+    outputs:
+    - contextPath: GitHub.TEAMMEMBER.ID
+      description: The ID of the team member.
+      type: Number
+    - contextPath: GitHub.TEAMMEMBER.Login
```

Comment:
* why are we using all upper-case letters for the context path? why not `TeamMember`?
* you are missing the `Team` output path

Reply:
You are absolutely right, no reason to use upper-case.. changed it.